app/javascript/admin/controllers/config_controller.js

```diff
@@ -139,14 +139,29 @@ export default class ConfigController extends Controller {
     const previewLogo = document.querySelector(
       '#logo-upload-preview .site-logo__img',
     );
+    const communityName = document.querySelector(
+      '.site-logo .site-logo__community-name',
+    );
 
     if (!previewLogo) {
       return;
     }
 
-    for (const logo of document.querySelectorAll('.site-logo__img')) {
-      if (logo !== previewLogo) {
-        logo.src = previewLogo.src;
+    // if we are showing the community name because this is the first time that the
+    // creator is uploading a logo, then we want it to replace the community name
```

Comment:
Quick, non-blocking suggestion: let's use "logo" instead of "it" here (if by "it" we do in fact mean "logo") so that this comment is less ambiguous!
```suggestion
    // creator is uploading a logo, then we want the logo to replace the community name
```